gems/pending/appliance_console/database_replication.rb

```diff
@@ -0,0 +1,116 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'English'
+
+module ApplianceConsole
+  class DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REPMGR_CONFIG = '/etc/repmgr.conf'.freeze
+
+    attr_accessor :cluster_name, :node_number,
+                  :primary_database, :primary_database_username, :primary_database_password,
+                  :primary_host, :standby_host
+
+    def ask_for_cluster_name
+      self.cluster_name = just_ask("cluster name", cluster_name)
+    end
+
+    def ask_for_node_number
+      self.node_number = ask_for_integer("node number")
```

Comment:
This should be more explicit. I would tell the user that this number identifies the node within the cluster and must be unique within this region.